ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -1302,7 +1303,23 @@ def get_storage_cluster(namespace=config.ENV_DATA["cluster_namespace"]):
         storage cluster (obj) : Storage cluster object handler
 
     """
-    sc_obj = OCP(kind=constants.STORAGECLUSTER, namespace=namespace)
+    # In FaaS platform, the StorageCluster CR is only present in the provider cluster
```

Comment:
Do you think it would make sense to move this information to the function's docstring? It seems to me important enough for the understanding of what the function is going to return.

Reply:
The function can be called irrrespictive of the cluster content. This change does not make a difference in the usage of the function.  There is not change in the return value of the object. Only 1 attribute of the OCS object is changed which will not affect the return expectation of the function.